Scripts/header.sh

```
@@ -0,0 +1,104 @@
+    y) year="$OPTARG" ;;
+    *) usage ;;
+  esac
+done
+
+# Check for mandatory arguments
+if [ -z "$directory" ] || [ -z "$creator" ] || [ -z "$company" ] || [ -z "$package" ]; then
+  usage
+fi
+
+# Define the header template
+header_template="//
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)